grafana/resource_cloud_stack.go

```diff
@@ -338,7 +338,7 @@ func waitForStackReadiness(ctx context.Context, d *schema.ResourceData) diag.Dia
 		return nil
 	}
 
-	err := resource.RetryContext(ctx, 2*time.Minute, func() *resource.RetryError {
+	err := resource.RetryContext(ctx, 5*time.Minute, func() *resource.RetryError {
```

Comment:
Would it make sense to make this a configurable value via an environment variable?

Reply:
Made it configurable as an attribute